### DISCUSSION TITLE
Does HYDRA have plans to support TLS_CLIENT_AUTH?

### DISCUSSION THREAD

**Discussion by WeWebsterchen** [ory/hydra#2529] [Q&A] [answered]
Does HYDRA have plans to support  client use TLS_CLIENT_AUTH?

**Comment by vinckr** [comment, marked as answer]
Do you mean [OAuth 2.0 Mutual-TLS Client Authentication and Certificate-Bound Access Tokens](https://datatracker.ietf.org/doc/html/rfc8705)?

**Reply by vinckr** [reply]
Hey @WeWebsterchen 
Just checking in if your question was answered. I did not find any open issue related to that feature, but you can open one with some more concrete implementation ideas!
I will mark this as answered for now, but feel free to come back to the discussion any time.

Thanks a ton! 🐝

**Reply by WeWebsterchen** [reply]
> Do you mean [OAuth 2.0 Mutual-TLS Client Authentication and Certificate-Bound Access Tokens](https://datatracker.ietf.org/doc/html/rfc8705)?

yes   when client register token_endpoint_auth_method set tls_client_auth_subject_dn.when client get access token verify client mTLS  cert match subject_dn.

**Reply by scthi** [reply]
Fyi.: There is [this](https://github.com/ory/hydra/issues/1084) open issue.
